src/SFA.Tl.Matching.Automation.Tests/Project/Framework/Helpers/PageInteractionHelper.cs

```diff
@@ -64,11 +94,28 @@ public static Boolean VerifyText(String actual, String expected)
                 + "\n Found: " + actual);
         }
 
+        public static Boolean VerifyText(By locator, int expected)
+        {
+            String expectedText = Convert.ToString(expected);
+            String actual = webDriver.FindElement(locator).Text;
+            
+            if (actual.Contains(expectedText))
+            {
+                return true;
+            }
+
+            throw new Exception("Text verification failed: "
+                + "\n Expected: " + expected
+                + "\n Found: " + actual);
+        }
+
```

Comment:
This method can be further simplified as below, as VerifyText with locator & string exist, that can be called after converting the int to String. Example below 

String expectedText = Convert.ToString(expected);
VerifyText(locator, expectedText);